proof-of-burn-project/contracts/Terms.sol

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: UNLICENSED
 
-pragma solidity ^0.8.0;
-import "./ownable.sol";
+pragma solidity ^0.6.0;
+import "./Ownable.sol";
 import "./ChainLinkOracle.sol";
 import "./Helper.sol";
 
-contract Terms is Ownable, Helper {
+contract Terms is Ownable, Helper, ChainLinkOracle {
```

Comment:
are we able to use the chain link oracle contract in the file without subclassing from it? just curious cause I didn't know if that was the right approach.